app/views/wikies/index.html.erb

```diff
@@ -80,7 +80,7 @@
 
 <% @allowed_collaborations.each do |wiky| %>
       <tr>
-        <td><%= wiky.title %></td>
+        <td><%= wiky.reload.title %></td>
```

Comment:
@LuisMenjivar Try this without the `.reload` call, so it is just `wiky.title`. Is the `reload` call needed?